theme/blog-btn.scss

```
@@ -24,6 +33,9 @@
   @media (max-width: 768px) {
     font-size: 12px;
     height: 28px;
+    // Match the shorter pill so the single line stays vertically centered
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)